movie_REST_API/movie_REST_API/App/SceneDelegate.swift

```diff
@@ -16,6 +16,7 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         window = UIWindow(frame: windowScence.coordinateSpace.bounds)
         window?.windowScene = windowScence
 
+        KeychainService().saveKey(KeychainService.Constants.key)
```

Comment:
```suggestion
        KeychainService().saveKey(KeychainService.Constants.key)
         let navigationController = UINavigationController()
         let assembleBuilder = AssembleBuilder()
         let router = Router(navigationController: navigationController, builder: assembleBuilder)
```
это все вынеси в метод